0_reporting_functions.R

```diff
@@ -377,27 +384,28 @@ ReportGeneration <- function(){
   portfolio_name_selectClean <- gsub("'","",portfolio_name_selectClean)
   
   
-  text$text <- gsub("start_year+10",start_year+10,text$text,fixed=T)
-  text$text <- gsub("start_year+5",start_year+5,text$text,fixed=T)
-  text$text <- gsub("start_year",start_year,text$text)
-  text$text <- gsub("investor_name_select",investor_name_selectClean,text$text)
-  text$text <- gsub("portfolio_name_select",portfolio_name_selectClean,text$text)
+  text$text <- gsub("Startyear+10",start_year+10,text$text,fixed=T)
+  text$text <- gsub("Startyear+5",start_year+5,text$text,fixed=T)
+  text$text <- gsub("Startyear+5",start_year+5,text$text,fixed=T)
+  text$text <- gsub("Startyear",start_year,text$text)
+  text$text <- gsub("InvestorName",investor_name_selectClean,text$text)
+  text$text <- gsub("PortfolioName",portfolio_name_selectClean,text$text)
   text$text <- gsub("SizeofPortfolio",paste0("\\\\$",reportdata$SizeofPortfolio),text$text)
   text$text <- gsub("TodaysDate",reportdata$TodaysDate,text$text)
   text$text <- gsub("ReportDate",reportdata$ReportDate,text$text)
   text$text <- gsub("PeerGroup",reportdata$PeerGroup,text$text)
   text$text <- gsub("AssetClass",reportdata$AssetClass,text$text)
   text$text <- gsub("AnalysisCoverage",reportdata$AnalysisCoverage,text$text)
-  text$text <- gsub("scenarioText",reportdata$scenarioText,text$text)
-  text$text <- gsub("scenarioName",reportdata$scenarioName,text$text)
-  text$text <- gsub("scenarioValue",reportdata$scenarioValue,text$text)
-  text$text <- gsub("scenarioDescription",reportdata$scenarioDescription,text$text)
-  text$text <- gsub("scenarioTemp",reportdata$scenarioTemp,text$text)
+  text$text <- gsub("ScenarioText",reportdata$scenarioText,text$text)
+  text$text <- gsub("ScenarioName",reportdata$scenarioName,text$text)
+  text$text <- gsub("ScenarioValue",reportdata$scenarioValue,text$text)
+  text$text <- gsub("ScenarioDescription",reportdata$scenarioDescription,text$text)
+  text$text <- gsub("ScenarioTemp",reportdata$scenarioTemp,text$text)
```

Comment:
not sure which naming conventions these are, but still worth a look

Reply:
see above

Reply:
Here for example: https://github.com/2DegreesInvesting/Templates/blob/77db076c87b5a136854d8a1a3d417ded1a3215fa/GeneralTemplateInput_v6_SB_en.tex#L255